### ISSUE TITLE
Client sends 0 page size

### ISSUE THREAD

**Issue by david4096** [ga4gh/ga4gh-server#1362]
When attempting to use the client against another GA4GH server it was found that it defaults to send a pageSize of 0. `{u'pageToken': u'', u'pageSize': 0}` are sent with the first request. This bug caused the client to continually request empty pages.

**Comment by jeromekelleher** [comment]
Surely this is a bug in the other GA4GH server? Sending a page size of 0 allows the server to decide what the size of the returned page is, making process slightly less inefficient.

**Comment by dcolligan** [comment]
Yeah, this sounds like a non-bug to me... we have `self._defaultPageSize = 100` in `backend.py` which the page size is set to if the client sends a page size of 0.

Closing...
